tests/integration_tests.rs

```diff
@@ -37,6 +37,12 @@ async fn test_lifecycle() {
         .await
         .expect("failed to create extension");
 
+    // pgmq meta was created
+    let _ = sqlx::query("'public.pgmq_meta'::regclass")
```

Comment:
I think this is needed?

```suggestion
    let _ = sqlx::query("select 'public.pgmq_meta'::regclass")
```